### PR TITLE
Added maven repackage to try to fix deployed application

### PR DESCRIPTION
**Pull request by hwilliams1**
Added maven repackage to try to fix deployed application

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -80,9 +81,20 @@
...
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-maven-plugin</artifactId>
+                <version>${spring-boot-maven-plugin.version}</version>
+                <configuration>
+                    <mainClass>${start-class}</mainClass>
+                </configuration>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>repackage</goal>
```
Comment (VaughanJackson123) [inline pom.xml:93]: I was coming to the same conclusion myself! Do we actually need to specify the start class though? I note they don't do that here: https://github.com/companieshouse/officer-delta-processor/blob/main/pom.xml.

Comment (hwilliams1) [reply pom.xml:93]: They do here https://github.com/companieshouse/search.api.ch.gov.uk/blob/master/pom.xml and this actually works in tilt and deploys properly

Comment (VaughanJackson123) [reply pom.xml:93]: Yes, sorry I see you do have a `start-class` property anyway.

Comment (VaughanJackson123) [reply pom.xml:93]: I should add `officer-delta-processor` does work in tilt and `cidev`.

**Pull request merged**